webview/src/setup/components/studio/CliIncompatible.tsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react'
-import { STUDIO_URL, SectionCollapsed } from 'dvc/src/setup/webview/contract'
+import { useDispatch } from 'react-redux'
+import { STUDIO_URL } from 'dvc/src/setup/webview/contract'
 import { EmptyState } from '../../../shared/components/emptyState/EmptyState'
 import { Button } from '../../../shared/components/button/Button'
+import { updateSectionCollapsed } from '../../state/webviewSlice'
 
-export const CliIncompatible: React.FC<{
-  setSectionCollapsed: (sectionCollapsed: SectionCollapsed) => void
-}> = ({ setSectionCollapsed }) => (
-  <EmptyState isFullScreen={false}>
-    <h1>DVC is currently unavailable</h1>
-    <p>
-      Locate DVC to connect to <a href={STUDIO_URL}>Studio</a>
-    </p>
-    <Button
-      text="Setup DVC"
-      onClick={() =>
-        setSectionCollapsed({
-          dvc: false,
-          experiments: true,
-          studio: true
-        })
-      }
-    />
-  </EmptyState>
-)
+export const CliIncompatible: React.FC = () => {
+  const dispatch = useDispatch()
+
+  return (
+    <EmptyState isFullScreen={false}>
+      <h1>DVC is currently unavailable</h1>
+      <p>
+        Locate DVC to connect to <a href={STUDIO_URL}>Studio</a>
+      </p>
+      <Button
+        text="Setup DVC"
+        onClick={() =>
+          dispatch(
+            updateSectionCollapsed({
```

Comment:
[Q] Is this how we handle this in other parts of the app (e,g plots)? Should we converge on a single method?

Reply:
Well, the plots' collapsed state is handled differently in the front end. Instead of having a section object, plots section state each have their own `isCollapsed` key. I suppose it makes sense that it was set up that way since, unlike setup, plots sections don't need to close/open other sections. 

We could possibly have both plots and setup webviews use a section collapsed object, making them more similar and allowing them to possibly share functions/methods 🤔